docs/source/chainerx/install/index.rst

```diff
@@ -14,14 +14,20 @@ Environment variables
 
 Configure the following environment variables before installing Chainer.
 
-========================== ================================================================================================
-Environment variable       Description
-========================== ================================================================================================
-``CHAINER_BUILD_CHAINERX`` ``1`` to build the ``chainerx`` package along with ``chainer``. ``0`` to skip. Default is ``0``.
-``CHAINERX_BUILD_CUDA``    ``1`` to build ``chainerx`` with CUDA support. ``0`` to skip. Default is ``0``.
-``CUDNN_ROOT_DIR``         Path to your cuDNN installation. Required when ``CHAINERX_BUILD_CUDA=1``.
```

Comment:
Shouldn't this explanation updated too since `CHAINERX_CUDNN_USE_CUPY` has higher priority?

Reply:
How do you think should it be updated? Just the order?

I don't think `CHAINERX_CUDNN_USE_CUPY` has a higer priority. It's just a "convenience" function for a limited number of audience.

Reply:
When `CHAINERX_CUDNN_USE_CUPY` is set it's not required any more so it should be noted.
I think currently most users would install cupy with chainerx so I feel it's a not limited thing.

Reply:
Currently ChainerX depends on CuPy only temporarily. I don't want to document as if it were a norm.
Also this option has unobvious pitfalls as written in the docs.

Reply:
As long as ChainerX is depending on CuPy I like the convenience function more since installing cuDNN is kind a difficult step in the setup.
And I think the checks in the cmake script is enough to know the reason:
https://github.com/chainer/chainer/blob/f8db3447e84634f29fbce37d5c253eed4cb1910f/chainerx_cc/CMakeLists.txt#L84-L105

Reply:
> And I think the checks in the cmake script is enough to know the reason:

Could you elabolate what this means?

I do think `CHAINERX_CUDNN_USE_CUPY` is not the "authentic" method that anyone can use. Do you agree with that?
The "authentic" one (`CUDNN_ROOT_DIR`) should come first IMO.